packages/integration/src/lib/workspace/workspace.state.ts

```diff
@@ -74,6 +81,14 @@ export class WorkspaceState implements OnDestroy {
       });
     });
 
+    this.actionFullExtent$$.push(this.featureActionsService.getWorkspaceFullExtentEmitter().subscribe(r => {
```

Comment:
Shouldn't this be the other way around? The action services should be stateless and hte "set full extent" action should modify/observe the workspace state.

Reply:
Could you take a look to this new version?